### DISCUSSION TITLE
Character planner

### DISCUSSION THREAD

**Discussion by totallynotaguynon** [LmeSzinc/StarRailCopilot#496] [Q&A] [answered]
Is character planner coming to english users somehow as well?
It is possible to plan through the website or asking if maybe manual imports are gonna be doable.

**Comment by LmeSzinc** [comment, marked as answer]
Yes, depending on when the oversea game client have the in-game planner added.

There are some planning websites but hard to use to be honest, you need to manual input your inventory while the in-game tool reads realtime data. If you somehow exported the results from planning websites, insert them into `config/<config_name>.json` under key `Dungeon.Planner`, examples:
(Key `item` and `total` are required, others will be updated during SRC runs)
```
      "Item_Raging_Heart": {
        "item": "Raging_Heart",
        "value": 20,
        "total": 65,
        "synthesize": 0,
        "time": "2020-01-01 00:00:00",
        "progress": 30.77,
        "eta": 1.9
      },
      "Item_Moon_Madness_Fang": {
        "item": "Moon_Madness_Fang",
        "value": {
          "purple": 12,
          "blue": 60,
          "green": 22
        },
        "total": {
          "purple": 154,
          "blue": 81,
          "green": 22
        },
        "synthesize": {
          "purple": 0,
          "blue": 0,
          "green": 0
        },
        "time": "2020-01-01 00:00:00",
        "progress": 18.78,
        "eta": 9.9
      },
```

**Reply by WakPd** [reply]
Planner actually in game for all of us now.
